packages/kit/src/module/install.ts

```diff
@@ -10,7 +11,12 @@ export async function installModule (moduleToInstall: string | NuxtModule, _inli
   const { nuxtModule, inlineOptions } = await normalizeModule(moduleToInstall, _inlineOptions)
 
   // Call module
-  const res = await nuxtModule(inlineOptions, nuxt) ?? {}
+  const res = (
+    isNuxt2()
+      // @ts-expect-error Nuxt 2 `moduleContainer` is not typed
+      ? await nuxtModule.call(nuxt.moduleContainer, inlineOptions, nuxt)
```

Comment:
(i am not sure about functions binding but i guess we were already bounding utils to this)